nuxeo/values.yaml

```diff
@@ -75,28 +75,38 @@ clid: ""
 mongodb:
   enabled: false
   protocol: mongodb
+  # deprecated since 3.1.0, use auth object instead
   # credentials:
-  # url:
-  # host:
+  host: ""
   port: 27017
+  # when set it overrides values from mongodb.host and mongodb.port
+  url: ""
+  auth:
+    enabled: false
+    username: ""
+    password: ""
+    # existing secret with MongoDB credentials (keys: mongodb-username, mongodb-password)
+    existingSecret: ""
   initContainer:
     repository: busybox
     tag: latest
     imagePullPolicy: IfNotPresent
 postgresql:
   enabled: false
-  # host:
+  host: ""
   port: 5432
-  # username:
-  # password:
+  username: ""
+  password: ""
+  # existing secret with PostgreSQL credentials (keys: postgresql-username, postgresql-password)
+  existingSecret: ""
```

Comment:
Should we deprecate these and introduce an auth object like MongoDB?

Reply:
I guess we don't need an `enabled` attribute for these settings. Good idea for GCS and S3!